component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPAuthenticator.java

```diff
@@ -168,6 +168,8 @@ protected void initiateAuthenticationRequest(HttpServletRequest request,
             showAuthFailureReasonOnLoginPage = Boolean.parseBoolean(parameterMap.get(
                     TOTPAuthenticatorConstants.CONF_SHOW_AUTH_FAILURE_REASON_ON_LOGIN_PAGE));
         }
+        Map<String, String> runtimeParams = getRuntimeParams(context);
```

Comment:
Lets move this code down so we will pull this from the context just before we need it